Program.cs

```diff
@@ -21,27 +22,57 @@ static void Main(string[] args)
             while (true)
             {
 
-                Console.WriteLine("1. Pobierz plik z internetu.");
+                Console.WriteLine("1. Wybierz plik wejściowy.");
                 Console.WriteLine("2. Zlicz liczbę liter w pobranym pliku.");
                 Console.WriteLine("3. Zlicz liczbę wyrazów w pliku.");
                 Console.WriteLine("4. Zlicz liczbę znaków interpunkcyjnych w pliku.");
-                Console.WriteLine("5. Zlicz liczbę zdań w pliku.");
+                Console.WriteLine("5. Zlicz liczbę zdań w pliku zakończonymi znakami interpunkcyjnymi.");
                 Console.WriteLine("6. Wygeneruj raport o użyciu liter (A-Z).");
                 Console.WriteLine("7. Zapisz statystyki z punktów 2-5 do pliku statystyki.txt .");
                 Console.WriteLine("8. Exit");
                 int menuOption = Convert.ToInt32(Console.ReadLine());
 
                 if (menuOption == 1)
                 {
-                    WebClient webClient = new WebClient();
-                    try
+                    Console.WriteLine("[T/N]");
+                    char choise = Convert.ToChar(Console.ReadKey());
+                    if (choise=='T')
                     {
-                        webClient.DownloadFile("https://s3.zylowski.net/public/input/5.txt?fbclid=IwAR1E-7Oh6St9e89VcEIQZ2zVyZhFoCY1fNK73fKvAtsanknNxUbbXVyCnMc", "5.txt");
+                        Console.WriteLine("Podaj ścieżkę pliku");
+                        string filepath = Convert.ToString(Console.ReadLine());
+                        Console.WriteLine("Podaj nazwę pliku");
```

Comment:
it is not required to use:
Console.WriteLine("Podaj nazwę pliku");
string namefile = Convert.ToString(Console.ReadLine());
it can be always downloaded to one file eg. "5.txt"